goal_src/jak2/pc/pckernel.gc

```diff
@@ -230,6 +230,11 @@
   (when (not (led-enabled? obj))
     (disable *led-fader-state*)
     )
+  
+   (if (-> *pc-settings* input-display?)
+        (input-display-on)
+        (input-display-off)
```

Comment:
Where are these functions defined?

Reply:
https://github.com/open-goal/jak-project/pull/3057/files#diff-80370092254773d4b5c4f8774df3cd686c9a1dad5528f790c727450d8d58f2d0R324

https://github.com/open-goal/jak-project/pull/3057/files#diff-80370092254773d4b5c4f8774df3cd686c9a1dad5528f790c727450d8d58f2d0R370

Reply:
Well that was less helpful than I wanted it to be, but they are in goal_src/jak2/pc/features/input-display.gc

Reply:
Oops they were hidden, thanks github.